### ISSUE TITLE
Map and road segments don't align, parts missing (v0.7.0.7) 

### ISSUE THREAD

**Issue by cwienands1** [sandermvanvliet/RoadCaptain#123]
I came across RoadCaptain as I'd like to construct a close to ideal route that covers all current Watopia segments. For that project I'll need to cover the new Coastal Road segment, too, so I was was very excited to see that mentioned in the 0.7.0.7 release notes. After I installed that version and started RoadCaptain, unfortunately this is what the map screen looked like:

![image](https://github.com/sandermvanvliet/RoadCaptain/assets/27843931/46d0ed68-3dc7-4b7e-8019-215d7a23e23d)

From what I can tell, there are two parts to the displayed map: A background graphic that now shows the coastal route, and then an overlay with the selectable segments. That second layer is missing the coastal road and is stretched vertically (y axis) to fill the same space as the underlying map graphic (that's presumably because it has less "height" due to the missing segment).

I had 0.7.0.6 installed before I initially installed 0.7.0.7. To make sure 0.7.0.7 didn't pick up any files left from 0.7.0.6, I uninstalled 0.7.0.7 completely and reinstalled it. Still same behavior. I know you mentioned "possibly rough around the edges", so I wanted to let you know about this issue. :-)

Thanks for the great work!

**Comment by sandermvanvliet** [comment]
Yeah this doesn’t look right at all…

I will investigate what went wrong here. Most likely some coordinates got upset.

**Comment by sandermvanvliet** [comment]
I've just downloaded the MSI from the last pre-release and installed it locally, just to make sure I'm looking at the right thing and the funny thing is that _it works on my machine_ (of course it does 🙄):
![image](https://github.com/sandermvanvliet/RoadCaptain/assets/2393265/b7d25021-18a0-403e-8faa-72181f2957e3)

Could you do me a favour and see if you can find a file `worlds.json` in the installation directory and if you can find it, attach it here?
Also, if you can: please check the file modification time of that file.

**Comment by cwienands1** [comment]
OK, I think I figured out the problem. It seems the installer does not overwrite updated files, and also doesn't remove them during uninstallation. I found the installation directory under _C:\Program Files (x86)\Codenizer BV\RoadCaptain_ and saved the attached _worlds.json_ file. Then I uninstalled and found that the entire directory still exists. So I removed it manually, reinstalled the application, and now the map and segments look and work correctly. All good on my end :-)
[worlds.json](https://github.com/sandermvanvliet/RoadCaptain/files/13801409/worlds.json)
